plugins/pangea/textGuard.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  HookEventType,
+  PluginContext,
+  PluginHandler,
+  PluginParameters,
+} from '../types';
+import { post, getText } from '../utils';
+import { VERSION } from './version';
+
+export const handler: PluginHandler = async (
+  context: PluginContext,
+  parameters: PluginParameters,
+  eventType: HookEventType,
+  options: {}
+) => {
+  let error = null;
+  let verdict = false;
+  let data = null;
+  try {
+    if (!parameters.credentials?.domain) {
+      throw Error(`'parameters.credentials.domain' must be set`);
+    }
+
+    if (!parameters.credentials?.apiKey) {
+      throw Error(`'parameters.credentials.apiKey' must be set`);
+    }
```

Comment:
Here, can remove it pls?

Reply:
I don't think so, cause there's no guardrail action happening. It's a pure usage issue, we could simply retrun `verdict` as `true`.

Reply:
@b4s36t4 So, is it confirmed as `true`? It seems that after the edit, you might want to say `false`.

Reply:
It's ok, will be good for now.